assets/sass/_container.scss

```diff
@@ -0,0 +1,48 @@
+#okta-sign-in.auth-container {
+
+  // Html and body styles in _base.scss that were moved here
+  height: 100%;
+  font-family: $fonts;
+  font-size: $font-size;
+  font-weight: $font-weight-body;
+  line-height: 1.4;
+  background-color: $secondary-bg-color;
+  min-width: 100%;
+  min-height: 100%;
+  color: $medium-text-color;
+  position: relative;
```

Comment:
Also I am wondering if this affects anything - if it doesn't change the appearance, we should remove it.